cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -22,6 +28,18 @@
     /usr/lib/@{multiarch}/libseccomp.so* mr,
     /lib/@{multiarch}/libseccomp.so* mr,
 
+    # normal libs in order biarch + usr-merged
+    /usr/lib64/libapparmor.so* mr,
+    /usr/lib64/libcap.so* mr,
+    /usr/lib64/libcgmanager.so* mr,
+    /usr/lib64/libdl-[0-9]*.so* mr,
+    /usr/lib64/libnih.so* mr,
```

Comment:
libnih is I think only used on 14.04 systems, you may not need those unless they are really present on solus.